index.js

```diff
@@ -1,12 +1,14 @@
 'use strict'
 
+require('rootpath')()
+
 var Hapi = require('hapi')
 var Primus = require('primus')
-var Joi = require('joi')
 var config = require('config')
 var Drone = require('./models/drone')
+var logger = require('lib/log.js')(module)
```

Comment:
I am unsure about this. If we make the system truly distributed we might want to think about
more sophisticated logging mechanisms then just ```debug```.
Disclaimer: I don't have a node js background (I work more with Java, C, Python etc.). Therefore, it might be that I am totally wrong about this one.